server/src/schema.py

```diff
@@ -26,18 +37,42 @@ class User(Base):
     team = relationship("Team", back_populates="members")
     team_name = Column(String, ForeignKey('teams.name'))
 
+    submissions = relationship("Submission", back_populates="user")
+
     def __repr__(self):
         return "<User(username='{}', name='{}', score={}, team='{}')>".format(self.username, self.name, self.score, self.team_name)
 
 
 class Challenge(Base):
-    __tablename__ = 'Challenges'
-    id = Column(Integer, autoincrement=True, primary_key=True)
+    __tablename__ = 'challenges'
+
+    identifier = Column(Integer, autoincrement=True, primary_key=True)
     value = Column(Integer)
     text = Column(String)
+    submissions = relationship("Submission", back_populates="challenge")
+
+    def __repr__(self):
+        return "<Challenge(id={}, value={}, text='{}')>".format(self.identifier, self.value, self.text)
+
+
+class Submission(Base):
+    __tablename__ = 'submissions'
+
+    identifier = Column(Integer, autoincrement=True, primary_key=True)
+    text = Column(String)
```

Comment:
what would this column represent?

Reply:
If it's what I think it is, maybe `description` is a better word?